fuel-client/Cargo.toml

```diff
@@ -21,9 +21,9 @@ clap = { version = "3.1", features = ["derive"] }
 cynic = { version = "1.0", features = ["surf"] }
 derive_more = { version = "0.99" }
 fuel-storage = "0.1"
-fuel-tx = { version = "0.10", features = ["serde"] }
+fuel-tx = { version = "0.11", features = ["serde"] }
```

Comment:
Consider upgrading to 0.12 instead

Reply:
I think I'll do that in a separate PR, just to reduce make the changes more isolated.